ccmlib/scylla_repository.py

```diff
@@ -187,9 +190,16 @@ def setup(version, verbose=True):
 
             scylla_arch_package_path = f'{scylla_product}-{scylla_arch}-package.tar.gz'
             scylla_noarch_package_path = f'{scylla_product}-package.tar.gz'
+            scylla_java_reloc = f'{scylla_product}-tools-package.tar.gz'
+            scylla_jmx_reloc = f'{scylla_product}-jmx-package.tar.gz'
 
             aws_files = aws_bucket_ls(s3_url)
 
+            if scylla_arch_package_path not in aws_files:
+                scylla_java_reloc = list(filter(re.compile(f'{scylla_product}-tools-[0-9].*.noarch.tar.gz').match, aws_files))[0]
```

Comment:
those are going to fail in a very not clear way, if the files would exists in the folder.
we might need to catch the IndexError,